### ISSUE TITLE
Need a way to specify the application timezone.

### ISSUE THREAD

**Issue by o-rod-1** [nexmo-se/opentok-secure-appointment#4]
Currently when I create a booking from Pacific time zone, it is one day behind.

**Comment by snexmo** [comment]
Agreed we need to either provide guidance on how timezone is handled (i.e. is it browser timezone or back-end timezone which takes precedence)? Or we need to implement timezone as a dropdown option.
